action/editcommit.php

```diff
@@ -118,6 +125,16 @@ public function handle_periodic_pull(Doku_Event &$event, $param) {
             $timeToWait = $this->getConf('periodicMinutes')*60;
             $now = time();
 
+			if (!empty($this->getConf('pingPushHost'))) {
+				// ping first
+				$retval = -1;
+				system("/usr/bin/timeout 0.15 ping -c 1 -W 0.1 " . $this->getConf('pingPushHost'), $retval);
```

Comment:
Here the parameter for the ping command is `ping` only - in line 89 it is `/bin/ping` - this should be consistent.
Would it make sense to spend another configuration for the ping push base command - e.g. `pingPushBaseCommand` with `/usr/bin/timeout 0.15 ping -c 1 -W 0.1` as default value ?

This would make it more generic and we don't have to mention that this will work on Linux only.